importer/api/__init__.py

```diff
@@ -144,6 +142,16 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    if element.tag == "video":
+        subname = get_video_sub(element)
+        if subname != "":
+            assets = StaticAsset.objects.filter(
+                course__id=resource.course_id,
+                path=subname,
+            )
+            for asset in assets:
+                resource.static_assets.add(asset)
```

Comment:
I'm looking at it on the command line, and it still doesn't look like it. I can `bulk_create`, but that actually creates. There's an `in_bulk` as well, but that retrieves. None of the other methods appear to be "`bulk_add`."